src/helpers/responsive-helpers.js

```diff
@@ -9,27 +9,14 @@ LICENSE file in the root directory of this source tree.
 import type {BreakpointsT} from '../styles/types.js';
 
 /**
- * Helper function that generates media queries based on given parameters
- *
- * E.g.
- * getMediaQuery({'max-width': '1280px', 'min-height': '720px'}, 'and') =>
- *   '@media screen and (max-width: 1280px) and (min-height: 720px)'
+ * Helper function that generates media queries based on breakpoint, e.g.
+ * getMediaQuery(720) => '@media screen and (min-width: 720px)'
  */
-export const getMediaQuery = (
-  options: {[string]: string},
-  booleanOperator: 'AND' | 'OR' = 'OR',
-): string => {
-  const mediaFeatureSeparator = booleanOperator === 'OR' ? ', ' : ' and ';
-
-  const mediaFeatures = Object.keys(options).map(
-    key => `(${key}: ${options[key]})`,
-  );
-
-  return `@media screen and ${mediaFeatures.join(mediaFeatureSeparator)}`;
-};
+export const getMediaQuery = (breakpoint: number): string =>
+  `@media screen and (min-width: ${breakpoint}px)`;
```

Comment:
I think we should keep it a media query builder vs a single `min-width` media query getter.

Reply:
My intention was to keep these two helpers consistent. Since getMediaQueries returns the media queries for breakpoints, which are min-width, then getMediaQuery should also return min-width for a single breakpoint.

Also, with the original implementation, it’s not guaranteed that the input is valid, unless we type the input options to valid CSS rules. And to make this truly generic, we would need to account for `all`/`print`/`screen`/`speech` media types, all media features, e.g. `any-hover`, `any-pointer`, `aspect-ratio`, etc., `not` and `only` logical operators, and any arbitrary combination of media types, features, and logical operators. Rather than try to implement the spec for media queries, I thought it would be simpler to limit the helper to supporting only breakpoints.

Let me know if this is ok!

Reply:
How about renaming it to `getMinWidthMediaQuery`? But as long as it's not a part of the public API it's ok to keep as is.

Reply:
My only issue with `getMinWidthMediaQuery` would be that we would also need to update the other method to `getMinWidthMediaQueries` to match.

I feel like the fact that it takes in a `breakpoint` argument already suggests that the return would be a min-width, since that is how we deal with breakpoints in the app..